main/systemd.go

```diff
@@ -38,17 +38,17 @@ func GetSystemdUnitFileInstallPath() string {
 
     if (err != nil) {
         errorMessage := fmt.Sprintf("Error getting distro name: %+v : %s", err, err.Error())
-	    errorLogger.Println(errorMessage)
+        errorLogger.Println(errorMessage)
     } else {
         infoLogger.Println(fmt.Sprintf("distro name: %s", distro))
     }
 
-    distros1 :=[]string{"ubuntu", "debian"}
-    distros2 :=[]string{"suse", "sle_hpc", "sles", "opensuse", "redhat", "rhel", "centos", "oracle"}
+    identicalSystemdPathDistros1 :=[]string{"ubuntu", "debian"}
+    identicalSystemdPathDistros2 :=[]string{"suse", "sle_hpc", "sles", "opensuse", "redhat", "rhel", "centos", "oracle"}
```

Comment:
I'm not sure these are much better... I was thinking more `debianDistros` vs `redhatDistros`.

Reply:
Honestly this list mix of all flavors, it's hard to come up with standard name. That's why I don't want to have specific name. I don't think 100% convinced with proposed names but if it looks better in overall sense than others, I'm good to change.

Reply:
@kevinclark19a Changed to proposed names